frontend_tests/node_tests/popovers.js

```diff
@@ -213,8 +212,6 @@ test_ui("actions_popover", (override) => {
 
     const target = $.create("click target");
 
-    override(popovers, "hide_user_profile", noop);
-
```

Comment:
This change should be in the commit removing the `hide_user_profile` call from `hide_all_except_sidebars` otherwise that commit will individually fail the node tests.

Reply:
Ah, missed it. Thanks